docs/guides/creatingasource.md

```diff
@@ -12,7 +12,10 @@ We will create:
 - [x] The `sockeye` target which serves as an event display.
 - [x] The `AWSSQSSource` which consumes events from an AWS SQS queue.
 
-## Event display
+!!! tip "Kubernetes namespace"
+    The Kubernetes namespace to contain these objects as well as the secret for AWS SQS credentials is up to your choice, and should be the same for all objects.
```

Comment:
IMO this somewhat long an complicated sentence would be equally clear if rewritten to:

> All objects mentioned in this guide must be created inside the same Kubernetes namespace.

I believe this wording makes the part about the choice of the namespace being up to the reader unnecessary, because "the same Kubernetes namespace" induces "any namespace". Might be very subjective, just a suggestion to keep things short and to the point.